solutions/remotemonitoring/scripts/all-in-one.yaml

```diff
@@ -560,6 +560,11 @@ spec:
             configMapKeyRef:
               name: deployment-configmap
               key: config.webservice.url
+        - name: PCS_SEED_TEMPLATE
```

Comment:
this section is for asa-manager-pod. Should put it in the config-pod.

Reply:
fixed